src/pages/gapsPatterns/GapsPatternsPage.tsx

```diff
@@ -176,6 +174,7 @@ const GapsPatternsPage = () => {
 
   return (
     <PageContainer>
+      <Title level={3}>{t('dashboard_page_graph_title').slice(0, -7)}</Title>
```

Comment:
Why `-7`? doesn't sound very robust.
What if someone will add / remove a character in the end of that string?